src/lib/io/sol-fs-storage.c

```diff
@@ -70,11 +73,14 @@ fs_write(const char *name, struct sol_buffer *buffer)
     return ret;
 }
 
-int
-fs_read(const char *name, struct sol_buffer *buffer)
+SOL_API int
+sol_fs_read_raw(const char *name, struct sol_buffer *buffer)
```

Comment:
modules shouldn't export `SOL_API`.  If multiple files of the same module use that function, just export it in the header, without the `SOL_API`.